Gemfile.lock

```diff
@@ -134,8 +134,8 @@ GEM
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.5.1)
```

Comment:
why does this file need to change at all here?

Reply:
it has some strange points but `bundle update webmock` emit this error. updating webmock & danger I can install webmock 3.5.

```
Bundler could not find compatible versions for gem "addressable":
  In Gemfile:
    cocoapods-core was resolved to 1.9.1, which depends on
      addressable (~> 2.6)

    danger was resolved to 5.3.0, which depends on
      octokit (~> 4.2) was resolved to 4.7.0, which depends on
        sawyer (>= 0.5.3, ~> 0.8.0) was resolved to 0.8.1, which depends on
          addressable (>= 2.3.5, < 2.6)

    webmock was resolved to 3.8.3, which depends on
      addressable (>= 2.3.6)
```

Reply:
Perhaps `bundle install` instead of `bundle update`?

Reply:
possibly it's unnecessary change. Gemfile refs to cocoapods-core at master and core needs addressable (~> 2.6) on current master but core 1.9.1 does not require it. I think this may cause bundle update error.

https://github.com/CocoaPods/Core/blob/01e301e2a30a312247252f0c155ec6b244f35c51/cocoapods-core.gemspec#L29

bundle update for webmock is required to run the added test case on ruby 2.6. ruby 2.6 is system ruby on Catalina and used in the latest machines in github actions and bitrise.  anyway, travis here uses ruby 2.0..? It might be better to remove the test case for ruby 2.6...

Reply:
I found disabling webmock in the testcase can avoid `unknown keyword: write_timeout` error without modifying Gemfile.lock. I have deleted the Gemfile.lock commit and force pushed. I think it should work.

How can this WebMock handling in the test case be better? How do you think of it?

Reply:
I think this resolved in https://github.com/CocoaPods/CocoaPods/pull/9776#discussion_r422530908